### PR TITLE
Add fields to api and db for first/last names

### PR DESCRIPTION
**Pull request by shawkinsl**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -92,9 +92,12 @@ def signup():
     username = request.form.get("username")
     password = request.form.get("password")
     email = request.form.get("email")
+    first_name = request.form.get("firstname")
+    last_name = request.form.get("firstname")
```
Comment (Tsintsir) [inline src/app.py]: typo? "lastname" instead of "firstname"

**Comment by shawkinsl** [comment]
lol fixed

**Comment by Tsintsir** [comment]
+1!

**Comment by sonph** [comment]
lgtm

**Pull request merged**